src/app/cart/cart.component.html

```diff
@@ -0,0 +1,18 @@
+<!-- <p>cart works!</p> -->
+<h2>Your Cart</h2>
+<div class="cart--item-list-container">
+    <ul class="item-list cart--item-list">
+        <li *ngFor="let item of this.getBindedChosenItems()">
+            <img
+                class="cart--item-icon"
+                src="{{'../../assets/icons/' + item.id + '.svg'}}"
```

Comment:
Shouldn't need to use relative urls for the images. We could have done `"assets/icons"+item.id+".svg"`